### ISSUE TITLE
preserve tabs

### ISSUE THREAD

**Issue by leeoniya** [leeoniya/reMarked.js#9]

**Comment by marijnh** [comment]
Only in code blocks, or everywhere?

**Comment by leeoniya** [comment]
yeah, for code only. this is obviously a non-starter for markup with non-significant leading whitespace (everything not `whitespace: pre*`)
